### PR TITLE
add prev-next plugin as per #475

### PR DESCRIPTION
**Pull request by jthegedus**
I saw #475 and thought I would get that done since it has a number of :+1:s

Add prev/next links at the footer of pages based on the sidebar. Supports left/right arrow keyboard navigation.

Feel free to close if an external plugin is more appropriate.

Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you are merging your commits to `master` branch.
* [x] Add some descriptions and refer relative issues for you PR.
* [x] DO NOT include files inside `lib` directory.

### CODE REVIEW

**Comment by timaschew** [comment]
Please run these commands to trigger netlify deployment for this Pull Request:
```sh
git commit --amend --no-edit
git push --force
```

**Comment by jthegedus** [comment]
Done! Great work pruning the repo issues :100:

**Comment by timaschew** [comment]
FYI: DNS has changed: https://deploy-preview-734--docsifyjs.netlify.com/#/

**Comment by timaschew** [comment]
Can you please rebase on origin/develop

File: src/plugins/prev-next.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+function getLink(link) {
```
[the remark this answers is not in the document]
Comment (timaschew) [reply src/plugins/prev-next.js]: Okay I see, you want to make it part of docsify itself.  
But it seems a bit weird copying the source code from the author here and just mention it in the docs:

> By [@davestewart](https://github.com/davestewart).

**Comment by timaschew** [comment]
This feature doesn't work as soon as you have anchors in the sidebar.

**Comment by jhildenbiddle** [comment]
FWIW, I would prefer to see this functionality incorporated via the existing Pagination plugin listed in the official docs: 

* https://docsify.js.org/#/plugins?id=pagination

The above plugin is also configuration using docsify-themeable: 

* https://jhildenbiddle.github.io/docsify-themeable/#/customization?id=pagination

I understand the argument for making pagination part of docsify core, but the same could be said for a number of plugins currently available. In the short term, I think continuing to keeping docsify core light and add functionality (search, tabs, pagination, etc.) via plugins is the right way to go. In the future (perhaps for the next major version update) we can review *all* plugins that we think are good candidates for adding to core and determine how to move forward. For example, we can reduce the number of requests by incorporating popular plugin functionality into core, but we'll have to evaluate how doing so affects the size of the library.

Just my $0.02. Really appreciate all the work @timaschew is doing to get the repo back in order and the contributions from folks like @jthegedus.

**Comment by timaschew** [comment]
Actually, ai totally agree with you.
So closing this issue.

**Pull request closed without merging**